### PR TITLE
Fix Order Item API & Added info about .env file in READEM.md

### PR DESCRIPTION
**Pull request by shivamag00**

### CODE REVIEW

File: src/components/Order/OrderItems.vue [hunk 1]
```diff
@@ -49,28 +49,20 @@ export default {
...
-                            pPrice:this.products[i].orderItems[j].product.price,
-                            pQuantity:this.products[i].orderItems[j].quantity
-                        })
-                    }
+          axios.get(`${this.baseURL}order/${this.orderID}?token=${this.token}`).then((response) => {
+                if(response.status === 200) {
+                  let j;
+                  this.product = response.data
+                  this.lengthofOrderItems = Object.keys(this.product.orderItems).length
+                  for(j=0;j<this.lengthofOrderItems;j++){
```
Comment (nilmadhab) [inline src/components/Order/OrderItems.vue]: use foreach loop, code becomes more readable

File: src/components/Order/OrderItems.vue [hunk 2]
```diff
@@ -49,28 +49,20 @@ export default {
...
+                    this.orderProducts.push({
```
Comment (nilmadhab) [inline src/components/Order/OrderItems.vue:57]: this.lengthofOrderItems.forEach((item)=> {
// process item
});

Comment (shivamag00) [reply src/components/Order/OrderItems.vue:57]: Done

File: src/components/Order/OrderItems.vue [hunk 3]
```diff
@@ -49,28 +49,19 @@ export default {
...
-                            pName:this.products[i].orderItems[j].product.name,
-                            pDescription:this.products[i].orderItems[j].product.description,
-                            pPrice:this.products[i].orderItems[j].product.price,
-                            pQuantity:this.products[i].orderItems[j].quantity
-                        })
-                    }
+          axios.get(`${this.baseURL}order/${this.orderID}?token=${this.token}`).then((response) => {
+                if(response.status === 200) {
+                  this.product = response.data
+                  this.lengthofOrderItems = Object.keys(this.product.orderItems).length
```
Comment (nilmadhab) [inline src/components/Order/OrderItems.vue:55]: is this.lengthofOrderItems variable required?

**Review by nilmadhab — APPROVED** [review]

**Pull request merged**